app/views/admin/users/show.html.erb

```diff
@@ -1,110 +1,57 @@
-    <section class="flex flex-col m:flex-row gap-4">
-      <div class="crayons-card p-3 s:p-4 m:p-7 flex-1">
-        <%= render "admin/users/show/notes/form" %>
-      </div>
-      <div class="crayons-card p-3 s:p-4 m:p-7 flex-1">
-        <%= render "admin/users/show/notes/index" %>
-      </div>
-    </section>
-  <% elsif @current_tab == "emails" %>
-    <%= render "admin/users/show/emails/verification" %>
-    <section class="flex flex-col m:flex-row gap-4">
-      <div class="crayons-card p-3 s:p-4 m:p-7 flex-1">
-        <%= render "admin/users/show/emails/form" %>
-      </div>
-      <div class="crayons-card p-3 s:p-4 m:p-7 flex-1">
-        <%= render "admin/users/show/emails/index" %>
-      </div>
-    </section>
-  <% elsif @current_tab == "reports" %>
-    <section class="crayons-card p-3 s:p-4 m:p-7">
-      <%= render "admin/users/show/reports/index" %>
-    </section>
-  <% elsif @current_tab == "flags" %>
-    <section class="crayons-card p-3 s:p-4 m:p-7">
-      <%= render "admin/users/show/flags/index" %>
-    </section>
-  <% end %>
-<% else %>
-  <div class="grid gap-6">
-    <div class="crayons-card p-6">
-      <h2 class="d-inline">
-        <% if @user.registered? %>
-          <%= @user.name %><%= link_to "@#{@user.username}", @user.path, class: "ml-2", target: "_blank", rel: "noopener" %>
-        <% else %>
-          @<%= @user.username %> has not accepted their invitation yet.
-        <% end %>
-      </h2>
-      <% if @user.registered? %>
-        <a href="<%= edit_admin_user_path(@user) %>" class="btn btn-primary float-right">Manage User</a>
-        <p class="font-italic">Member since <%= @user.created_at.strftime("%b %e '%y") %></p>
-      <% end %>
-
-      <hr>
-
-      <h2>General Info</h2>
-      <dl>
-        <dt>ID:</dt>
-        <dd><%= @user.id %></dd>
-        <dt>Username:</dt>
-        <dd><%= @user.username %></dd>
-        <dt>Name:</dt>
-        <dd><%= @user.name %></dd>
-        <dt>Email:</dt>
-        <dd><%= @user.email %></dd>
-        <dt>Access Locked:</dt>
-        <dd><%= @user.access_locked? %></dd>
-        <dt>Twitter:</dt>
-        <% if @user.twitter_username %>
-          <dd><%= link_to @user.twitter_username, "https://twitter.com/#{@user.twitter_username}" %></dd>
-        <% else %>
-          <dd>N/A</dd>
-        <% end %>
-        <dt>GitHub:</dt>
-        <% if @user.github_username %>
-          <dd><%= link_to @user.github_username, "https://github.com/#{@user.github_username}" %></dd>
-        <% else %>
-          <dd>N/A</dd>
-        <% end %>
-      </dl>
+  </div>
+<% elsif @current_tab == "notes" %>
+  <section class="flex flex-col m:flex-row gap-4">
+    <div class="crayons-card p-3 s:p-4 m:p-7 flex-1">
+      <%= render "admin/users/show/notes/form" %>
     </div>
-
-  <% if @user.registered? %>
-    <%= render "activity" %>
-    <%= render "credits" %>
-    <%= render "notes" %>
-    <%= render "user_organizations" %>
-    <%= render "email_tools" %>
```

Comment:
@juliannatetreault in addition we should probably:
1. In addition to the _email_tools, we should look at which of these partials are no longer being used (since they were on the `edit` page) and delete them as well. I have not double checked this but I  don't think any of them are being used any longer.
2. In addition, we should delete the `/admin/users/<id>/edit` route and double check that all references to it are changed to the `/admin/users/<id>`.
3. Disable and Remove the feature flag through a DUS.

Reply:
> In addition to the _email_tools, we should look at which of these partials are no longer being used (since they were on the edit page) and delete them (and their specs) as well. I have not double checked this but I don't think any of them are being used any longer.

Agreed! I was actually going to do this in a separate PR, but can tackle that in this PR if you'd prefer. 😊

> In addition, we should delete the /admin/users/<id>/edit route and double check that all references to it are changed to the /admin/users/<id>.

Great catch! I actually saw that something, somewhere mentioned some wonkiness with the routes recently, so I will see if this was already resolved and if not, will take care of it in this PR!

> Disable and Remove the feature flag through a DUS.

@Ridhwana, so, this is something I wanted to ask you about: Do we want to remove this feature flag in this PR or do we want to reuse it for the Member Detail Index view work (similar to what we've decided with the `admin_creator_settings` feature flag)? I'm fine with either. 😊

Reply:
Feel free to split up the PR the way you see fit. I don't mind either way :) 

I think we need to disable the feature flag because if we use that flag again for in progress development work, and it's automatically enabled, it will then show that feature to all Forems. 

This was an oversight of mine with the creator_onboarding for the same reason as above. I plan on adding a DUS to disable it sometime this week.

I think for the admin member view let's disable and remove it and we can create an admin_index_view one specific for that feature, since the index view is not a v2 or extension of the member view like with creator setup and onboarding.

Reply:
> Feel free to split up the PR the way you see fit. I don't mind either way :)

Will do! 😊 

> I think for the admin member view let's disable and remove it and we can create an admin_index_view one specific for that feature, since the index view is not a v2 or extension of the member view like with creator setup and onboarding.

Sounds great and done (minus the part where you mentioned creating an `admi_index_view` flag 😄 )! ✔️